src/components/color-picker.tsx

```diff
@@ -134,26 +154,28 @@ export function ColorPicker({ color, onChange }: ColorPickerProps) {
               />
             </div>
 
+            {/* Discord preset swatches */}
             <div className="grid grid-cols-6 gap-2">
-              {discordColors.map((discordColor, index) => (
+              {discordColors.map((c, i) => (
```

Comment:
```suggestion
              {discordColors.map((c) => (
```

'i' is defined but never used so its better to remove it